client/Lykke.Service.PayInternal.Client/Models/Transfer/TransferStatusError.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Lykke.Service.PayInternal.Client.Models.Transfer
+{
+    /// <summary>
+    /// Transfer status error. If Transfer status is error, the property describe why. If status is not Error - NotError present.
+    /// </summary>
+    public enum TransferStatusError
+    {
+        /// <summary>
+        /// No errors. If status of transaction request is not Error.
+        /// </summary>
+        NotError = 0,
```

Comment:
If you make it public then I'd better use None. It if is internal then ok, but visibility has to changed to internal.